torch/_dynamo/utils.py

```diff
@@ -1645,6 +1645,8 @@ def to_numpy_helper(value):
 
 def numpy_to_tensor(value):
     """Convert tnp.ndarray to tensor, leave other types intact. If a list/tuple, loop through it to convert."""
+    if isinstance(value, np.ndarray):
+        return torch.as_tensor(value)
```

Comment:
Better to remove this one, as, if our logic is correct, an `np.ndarray` should never get to this point. We should probably even assert `not isinstance(value, np.ndarray)`

Reply:
Let me double check, I am pretty certain we do hit this, and that is why I added it. Why would you expect us to not get here?